source/time.tex

```diff
@@ -9507,9 +9486,9 @@
 
 \pnum
 \effects
-Constructs a \tcode{zoned_time} by
-initializing \tcode{zone_} with \tcode{std::move(z)} and
-default constructing \tcode{tp_}.
+Initializes
+\tcode{zone_} with \tcode{std::move(z)} and
+default-constructs \tcode{tp_}.
```

Comment:
I see that above, a occurrence in numerics.tex uses a dash. Just below, you don't. I also remember one of the "Mandating the Library" papers not using a dash for such a change.

Reply:
We use "default-constructed" as a hypenated adjective phrase, but verb usages like this one should be unhyphenated "default constructs" or (IMO better) "default-initializes" which is a hyphenated word of power.

Reply:
Thanks, fixed.